clickable.json

```diff
@@ -0,0 +1,158 @@
+{
+    "builder": "custom",
+    "build": "echo Run \\'clickable build-libs --debug\\' to build && false",
```

Comment:
The instructions above do not include the `--debug` flag. If it is not needed, it should be removed in this help message as well.

Reply:
True, --debug should be on so debugging symbols and the QML debugger are enabled.